lib/constants.go

```diff
@@ -911,9 +917,24 @@ func (params *DeSoParams) EnableRegtest() {
 	// Set the PoS default jail inactive validator grace period epochs to 3.
 	params.DefaultJailInactiveValidatorGracePeriodEpochs = 3
 
+	// Check for consensus transition every second.
+	params.FastHotStuffConsensusTransitionCheckDuration = 1 * time.Second
```

Comment:
I feel like I know exactly what this is doing because I've run the PoW->PoS cutover so many times but... where is it actually being used? I don't see where it's getting used here.

Reply:
Ok I see it's in `resetFastHotStuffConsensusTransitionCheckTime`. Looks good. Man, I *really* wish I had this when building the AMM haha. This is literally going to make all of my tests so much better.